testing/A_introduction.md

```diff
@@ -22,6 +22,8 @@ Finished in 0.2 seconds (0.2s on load, 0.00s on tests)
 
 We already have four tests!
 
+> Note: We don't need to create or migrate our test database because the test helper takes care of all that for us. If your tests fail to run and the error is related to Postgres, see [the section on ecto.create in the Mix Tasks guide](http://www.phoenixframework.org/docs/mix-tasks#section--ecto-create-) for more information.
```

Comment:
@wsmoak, thanks so much for this! Very much appreciated. :)

Could I ask one small favor? Please change the "your" to "the". That would become: "If the tests fail . . . ". With that and a rebase to squash the commits, we'll be all set!

(As much as possible, we try to avoid the use of "you" and "your" in the guides in favor of "we" and "our". My hope is that it gives readers a sense that we're all in it together. :) ) 

Thank you!